src/core/on-chain/index.ts

```diff
@@ -655,7 +652,16 @@ export const OnChainMixin = (superclass) =>
                 transactionType: "receipt",
                 onUs: false,
               })
+
               onchainLogger.info({ success: true, ...metadata })
+
+              await transactionNotification({
+                type,
+                user: this.user,
+                logger: onchainLogger,
+                amount: sats,
+                txid: matched_tx.id,
+              })
```

Comment:
I add the notification here because is the safest place to put it, block handler some times is triggering twice per block (at least in dev environment).

Reply:
also in prod